code/modules/reagents/chemistry/holder.dm

```diff
@@ -333,9 +333,9 @@ var/const/INGEST = 2
 			od_chems.Add(R.id)
 	return od_chems
 
-/datum/reagents/proc/process()
+/datum/reagents/process()
```

Comment:
This thing right here.

Don't get me started. IT exists purely so 2 reagents can heat/cool other reagents over time. Because this exists, every single reagent container in the entire game ends up processing because of it. It bloats up SSobj to like 2-3 times the size it would be.

I'm tempted to kill it off after this gets merged, because this is disgusting.

Reply:
What the actual fuck lmao

Reply:
Yeah, stable mutagen uses this too.

I'm sure there's a lot better way of handling it, and once I figure it out, this is going to go, because this is gross.

Reply:
Far out thinking went down to get this to work, further out thinking is required to refactor it. Oh God